components/molecules/modals/CongratsModal.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import { StyleSheet } from 'react-native';
+import Button1 from '../../atoms/buttons/Button';
+import { Text, View } from '@gluestack-ui/themed';
+
+const CongratsModal = (props) => {
+    return (
+        <View>
+             <View style={styles.confirmationCard}>
+            <Text style={styles.confirmationText}>Congratulation🎉</Text>
```

Comment:
Need to use FormattedMessage tag from react-intl everywhere where there is text hardcoded in
https://formatjs.io/docs/react-intl/components/#usage


Example:
```js
import { FormattedMessage } from 'react-intl';
.
.
.
<Text style={styles.confirmationText}>
  <FormattedMessage id="congratsModal.congratulation" defaultMessage="Congratulation🎉" />
</Text>
```